build.gradle

```diff
@@ -69,8 +71,42 @@ jar {
     }
 }
 
-processResources
-{
+import org.ajoberstar.grgit.Grgit
+
+task contributors {
+    group = 'other'
+    description = 'Creates a contributors file based on the git commit history'
+
+
+    doLast {
+	def header = '''
```

Comment:
Instead of deleting credits.txt and putting this text here, could we not just have credits.txt contain a ${CONTRIBUTORS} variable that gets replaced? Think of how version numbers are populated in mcmod.info